pkg/auditlog/storage/v2/admin_audit_log.go

```diff
@@ -41,6 +43,7 @@ var (
 
 type AdminAuditLogStorage interface {
 	CreateAdminAuditLogs(ctx context.Context, auditLogs []*domain.AuditLog) error
```

Comment:
The `CreateAdminAuditLogs` will not be used after apply this PR, maybe we can remove it?

Reply:
I'm not sure yet. So, I'm keeping it for now.